inst/COPYRIGHTS

```diff
@@ -56,54 +56,6 @@ along with RAVE.  If not, see <http://www.gnu.org/licenses/>.
 
 ====
 
-libiris2odim is written Mark Couture, Environment Canada, with contributions
```

Comment:
these copyrights should be kept on the main branch

Reply:
reinserted